okhttp/src/commonMain/kotlin/okhttp3/Call.kt

```diff
@@ -15,14 +15,14 @@
  */
 package okhttp3
 
-import java.io.IOException
+import okio.IOException
 import okio.Timeout
 
 /**
  * A call is a request that has been prepared for execution. A call can be canceled. As this object
  * represents a single request/response pair (stream), it cannot be executed twice.
  */
-interface Call : Cloneable {
+interface Call {
```

Comment:
kotlin.Cloneable doesn't work in common, and it's purely a marker interface, so while an API change, the actual implementations will all implement this, see RealCall.